pkg/interop/doc.go

```diff
@@ -8,14 +8,17 @@ in the documentation of respective functions.
 
 Types defined here are used for proper manifest generation. Here is how Go types
 correspond to smartcontract and VM types:
+
 	int-like - Integer
 	bool - Boolean
 	[]byte - ByteArray (Buffer in VM)
 	string - String (ByteString in VM)
 	(interface{})(nil) - Any
 	non-byte slice - Array
 	map[K]V - map
+
 Other types are defined explicitly in this pkg:
+
 	Hash160, Hash256, Interface, PublicKey, Signature
```

Comment:
Move it out of the code and use doc links?

Reply:
I'm not sure about doc lincs. The documentation says:

> Doc links are links of the form “[Name1]” or “[Name1.Name2]” to refer to exported identifiers in the current package

But Goland doesn't recognize the `[Hash160]` expression. If I use `[interop.Hash160]`, the link is there.

Reply:
Maybe Goland is not up to date wrt this? We're in the `interop` package here, so I'd suppose `[Hash160]` is OK. Can we try rendering it with Go tools? It's not critical though, can be changed later if anything is wrong.

Reply:
Yes, and `go doc` renders these links properly, so let's keep `[Hash160]` form. Fixed.